yew-macro/src/props/element.rs

```diff
@@ -1,19 +1,24 @@
 use super::{Prop, Props, SpecialProps};
 use lazy_static::lazy_static;
+use proc_macro2::Span;
 use std::collections::HashSet;
 use std::iter::FromIterator;
 use syn::parse::{Parse, ParseStream};
+use syn::spanned::Spanned;
 use syn::{Expr, ExprTuple};
 
 pub enum ClassesForm {
-    Tuple(Vec<Expr>),
-    Single(Box<Expr>),
+    Tuple(Span, Vec<Expr>),
```

Comment:
We can make this much prettier by passing `ExprTuple` directly instead of `Vec<Expr>`. That way we don't need to have a separate field for the span.

Reply:
That's genius!

Fixed in e439e6e